src/services/emptyDocuments.ts

```diff
@@ -96,6 +96,7 @@ const emptyDocuments: Document[] = [
     eventStatus: "CLOSED",
     preResources: [],
     postResources: [],
+    eventSurvey: [],
```

Comment:
Why is eventSurvey an array? Will there be multiple surveys for each event?

Reply:
There wouldn't be multiple surveys, however. would give the option incase they wanted to send more than one survey eventually?

Reply:
If this isn't the case, what would replace it?